stock_quant_history/readme/DESCRIPTION.rst

```diff
@@ -7,3 +7,5 @@ are present in the database.
 
 Next snapshot is computed based on the previous snapshot present in the database.
 
+Once a snapshot has been taken, every locked and done picking won't be able to unlock
+to avoid incoherent stock history.
```

Comment:
While creating new snapshot should re-lock un-lock picking ?

I believe it should be at least in the in the Roadmap, there is nothing to force user to re-lock picking once it has been unlocked.